packages/sanity/src/desk/comments/src/components/pte/comment-input/Editable.tsx

```diff
@@ -199,49 +173,57 @@ export function Editable(props: EditableProps) {
     }
   }, [value, selection])
 
+  // Update the mentions search term in the mentions menu
+  useEffect(() => {
+    mentionsMenuRef.current?.setSearchTerm(mentionsSearchTerm)
+  }, [mentionsSearchTerm])
+
+  // Close mentions if the user selects text
+  useEffect(() => {
+    if (mentionsMenuOpen && selection && !isEqual(selection.anchor, selection.focus)) {
+      closeMentions()
+    }
+  }, [mentionsMenuOpen, closeMentions, selection])
+
+  const popoverContent = (
+    <MentionsMenu
+      inputElement={editableRef.current}
+      loading={mentionOptions.loading}
+      onSelect={insertMention}
+      options={mentionOptions.data || EMPTY_ARRAY}
+      ref={mentionsMenuRef}
+    />
+  )
+
   return (
-    <>
-      <PortalProvider element={rootElementRef.current}>
-        <StyledPopover
-          arrow={false}
-          constrainSize
-          content={
-            <MentionsMenu
-              ref={mentionsMenuRef}
-              loading={mentionOptions.loading}
-              onSelect={insertMention}
-              options={mentionOptions.data || []}
-              inputElement={editableRef.current}
-              // error={mentionOptions.error}
-            />
-          }
-          disabled={!mentionsMenuOpen}
-          fallbackPlacements={['bottom', 'top']}
-          open={mentionsMenuOpen}
-          placement="bottom"
-          portal
-          ref={setPopoverElement}
-          referenceElement={cursorElement}
-        />
-      </PortalProvider>
-
-      <EditableWrapStack ref={rootElementRef} data-ui="EditableWrapStack">
-        <PortableTextEditable
-          data-testid="comment-input-editable"
-          data-ui="EditableElement"
-          onBeforeInput={onBeforeInput}
-          onBlur={onBlur}
-          onFocus={onFocus}
-          onKeyDown={handleKeyDown}
-          ref={editableRef}
-          renderBlock={renderBlock}
-          renderChild={renderChild}
-          renderPlaceholder={renderPlaceholder}
-          selection={initialSelectionAtEndOfContent}
-          style={INLINE_STYLE}
-          tabIndex={focusLock ? 0 : undefined}
-        />
-      </EditableWrapStack>
-    </>
+    <div ref={rootElementRef}>
+      <StyledPopover
```

Comment:
We previously used a portal to render the `Popover` with the`rootElementRef` element as the portal element. However, this is unnecessary, and we can just place the `Popover` directly inside the `rootElementRef` without using a portal (the same result).